internal/testrunner/runners/system/runner.go

```diff
@@ -1285,7 +1675,7 @@ func writeSampleEvent(path string, doc common.MapStr, specVersion semver.Version
 		return fmt.Errorf("marshalling sample event failed: %w", err)
 	}
 
-	err = os.WriteFile(filepath.Join(path, "sample_event.json"), body, 0644)
+	err = os.WriteFile(filepath.Join(path, "sample_event.json"), body, 0o644)
```

Comment:
Nit. These are equivalent, right? :thinking: I would leave it as it was, as this is the syntax we are using in other places.
```suggestion
	err = os.WriteFile(filepath.Join(path, "sample_event.json"), body, 0644)
```

Reply:
Sure!

I've started to try `nvim` with the LSP configuration, and it does those changes... I need to check how to disable this one specifically.